src/main/java/gov/nasa/pds/tools/validate/content/array/ArrayContentValidator.java

```diff
@@ -252,6 +252,16 @@ private void validatePosition(Array array, ArrayObject arrayObject, ArrayLocatio
           LOG.warn("validatePosition:Unhandled dataType {}", dataType);
           break;
       }
+    } catch (ArrayIndexOutOfBoundsException aiobe) {
+      String message = "No message specified, but probably index out of range. Report this message to have it fixed (have original exception given a message";
+      if (aiobe.getMessage() != null)
+        message = aiobe.getMessage();
+      throw new IOException(message);
+    } catch (IllegalArgumentException iae) {
+      String message = "No message specified, but probably wrong number of dimensions. Report this message to have it fixed (have original exception given a message";
```

Comment:
Ditto. (Extreme pedants are the worst! 😆)

Reply:
fixed